lib/Factory/DisplayFactory.php

```diff
@@ -605,8 +605,9 @@ public function query($sortOrder = null, $filterBy = [])
 
         $limit = '';
         // Paging
-        if ($filterBy !== null && $parsedBody->getInt('start') !== null && $parsedBody->getInt('length') !== null) {
-            $limit = ' LIMIT ' . $parsedBody->getInt('start', ['default' => 0]) . ', ' . $parsedBody->getInt('length', ['default' => 10]);
+        if ($parsedBody->hasParam('start') && $parsedBody->hasParam('length')) {
+            $limit = ' LIMIT ' . $parsedBody->getInt('start')
```

Comment:
This parameter could be present but not set - i.e. `start=&length=10` which I think would fail here because getInt would return `null` ?

Reply:
We should always have defaults via gridRenderFilter, but I can double check, this is basically the same change you made to displayGroupFactory in the past - but we can certainly have default value there too, just in case.